mep/settings.py

```diff
@@ -55,6 +55,7 @@
     'wagtail.core',
     'wagtail.contrib.redirects',
     'taggit',
+    'fixture_magic',
```

Comment:
This change shouldn't have been checked in and appears to be what's breaking the build. Just pushed a commit to remove it.

Reply:
ah, thank you - I tried to put it in `optional_apps`, but it wasn't working while it was there, so had to do this temporarily and forgot about it.